src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Doctrine;
+
+use Doctrine\DBAL\Connection as DriverConnection;
+use Doctrine\DBAL\Exception\TableNotFoundException;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\Synchronizer\SingleDatabaseSynchronizer;
+use Doctrine\DBAL\Types\Type;
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
+/**
+ * @author Vincent Touzet <vincent.touzet@gmail.com>
+ */
+class Connection
+{
+    /**
+     * Configuration of the connection.
+     *
+     * * table_name : name of the table
```

Comment:
```suggestion
     * * table_name: Name of the table.
```

(same for the other ones)